src/tabs/implementation.ts

```diff
@@ -41,8 +41,10 @@ function openTab(
       );
       if (existingTab) {
         chromeApi.tabs.highlight({
+          windowId: existingTab.windowId,
           tabs: existingTab.index
         });
+        chromeApi.windows.update(existingTab.windowId, {focused: true});
```

Comment:
Would you mind running Prettier to update the formatting here? You should be able to run `yarn prettier:fix`.

Apologies, I hadn't yet configured CircleCI to run linters & tests on PRs. It should be triggered the next time you merge another commit :)

Reply:
Done.